downstream/modules/catalog/proc-preparing-worker-installer.adoc

```diff
@@ -76,13 +76,14 @@ servicescatalog_controller_name: Marketing
 
 servicescatalog_controller_user: marketing_sa
 servicescatalog_controller_password: password
-====
+-----
++
 . Save each file
 . Open the `inventory` file in the {PlatformNameShort} installer:
 +
-====
+-----
 # vi inventory
-====
+-----
 +
 . Add the `host_vars` for each host under `[servicecatalog_workers]`
```

Comment:
Suggestion: Add a period at the end of the sentence.

Reply:
@cbudz Aside from 3 minor suggestions. LGTM.

Reply:
@lmalivert I've implemented your suggestions here and updated the preview.

Reply:
@cbudz LGTM. Ready to merge.